MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/GiniMerchant.swift

```diff
@@ -85,12 +85,12 @@ public struct DataForReview {
     public init(id: String, 
                 secret: String,
                 domain: String,
-                apiVersion: Int = Constants.defaultVersionAPI,
+                apiVersion: Int = Constants.merchantVersionAPI,
                 logLevel: LogLevel = .none) {
         let client = Client(id: id, secret: secret, domain: domain, apiVersion: apiVersion)
-        self.giniApiLib = GiniHealthAPI.Builder(client: client, logLevel: logLevel.toHealthLogLevel()).build()
+        self.giniApiLib = GiniHealthAPI.Builder(client: client, api: .merchant, logLevel: logLevel.toHealthLogLevel()).build()
         self.documentService = DefaultDocumentService(docService: giniApiLib.documentService())
-        self.paymentService = giniApiLib.paymentService()
+        self.paymentService = giniApiLib.paymentService(apiDomain: APIDomain.merchant, apiVersion: apiVersion)
```

Comment:
[Clarification]: Why do we need to pass `apiDomain` and `apiVersion` in `paymentService` directly if we already did it in line 91 for `apiLib` which was used for the creation of `paymentService`?

Reply:
It's only because PaymentService it's optional and in case it's not found, there is a default initializion in case is found nil:

`public func paymentService(apiDomain: APIDomain = .default, apiVersion: Int = Constants.defaultVersionAPI) -> PaymentService {
        return payService ?? PaymentService(sessionManager: SessionManager(userDomain: .default), apiDomain: apiDomain, apiVersion: apiVersion)
    }`